### PR TITLE
updated python3 to 3.8.10

### LINKED ISSUE #2438: win32: Update Python to 3.8
_[It's time.](https://www.youtube.com/watch?v=AiBqyXNtOEs&t=1m18s)_

### PR DESCRIPTION
**Pull request by DjLegolas**
Switched to Python 3.8.2 and resolved some `FIXME` sections in python plugin.

should close #2438.

Note:
AppVeyor must be switched to python 3.8.2 for it to pass.

### CODE REVIEW

File: plugins/python/_hexchat.py [hunk 1]
```diff
@@ -220,11 +219,10 @@ def ptr_getter(field):
     return ret
 
 
-# TODO: 'command' here shadows command above, and should be renamed to cmd
-def hook_command(command, callback, userdata=None, priority=PRI_NORM, help=None):
```
Comment (TingPing) [inline plugins/python/_hexchat.py:224]: I appreciate handling the TODOs but in Python these are kind of public API and can't be changed.

Comment (TingPing) [reply plugins/python/_hexchat.py:224]: Lets just keep this MR entirely to the win32 build changes.

Comment (DjLegolas) [reply plugins/python/_hexchat.py]: Sorry for the (very) long delay in response. 😅 
Removed all TODOs related code in this MR.

**Comment by DjLegolas** [comment]
Updated to latest version of Python 3.8 for Windows which has an installer (3.8.10).
In addition, removed all `FIXME`/`TODO` fixes from this MR.

**Comment by TingPing** [comment]
Thanks, you still left whitespace changes to `src/common/dbus/example.py` which are unreleated.

**Comment by DjLegolas** [comment]
> Thanks, you still left whitespace changes to `src/common/dbus/example.py` which are unreleated.

whoops, forgot about this one. fixed.

**Comment by crackwitz** [comment]
can you please make an installer that contains this update? the last release, 2.16.1, is from Feb 2022 and still demands Python 3.6

**Comment by TingPing** [comment]
@crackwitz Here is an installer https://github.com/hexchat/hexchat/actions/runs/3401046512

Help appreciated updating the Windows builds.

**Pull request merged**